src/data/invalid/MagsAnalysisActivity-invalid.yaml

```diff
@@ -0,0 +1,22 @@
+#invalid because uses text in the ncbi_lineage_taxIDs slot
```

Comment:
please add that to the filename

Reply:
I thought we decided that a comment was sufficient since adding descriptions to filenames can make them really long.

Reply:
I did in parallel update the filename.